RTech.CoffeeMachine.Api/appsettings.json

```diff
@@ -5,6 +5,6 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "WeatherProviderBaseAddress": "https://api.open-meteo.com"
```

Comment:
That pretty much sums up the PR review.
I usually like to add my own commentary in PR's to help the reviewer.
Thank you for reading & please don't hesitate to contact me if you have any questions, cheers.